packages/ecc-client-ga4gh-wes/src/components/wes-run/definition/wesRun.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  FASTElement,
+  customElement,
+  attr,
+  observable,
+} from '@microsoft/fast-element';
+import template from './wesRun.template.js';
+import styles from './wesRun.styles.js';
+import WorkflowData from './workflowData.js';
+import { cancelWorkflow, fetchWorkflow } from '../../../data/index.js';
+
+@customElement({
+  name: 'ecc-client-ga4gh-wes-run',
+  template,
+  styles,
+})
+export default class WESRun extends FASTElement {
+  @attr baseURL = '';
+
+  @attr id = '';
+
+  @attr state = '';
+
+  @observable isLoading = true;
+
+  @observable data: WorkflowData = {
```

Comment:
this makes the class unnecessarily long, create a template (maybe in a separate file for templates if that makes sense) and assign this to it. And if this is going to be filled automatically, is there any need to list its fields, maybe just make the data type Workflow or null.

Reply:
will do if code duplicacy occurs, as of now, moving just a small chunk of code makes no sense as this is just a small module of a bigger component `ecc-wes-runs` .

Reply:
fair enough, but you don't need to list out all the properties for this object, you can just leave it empty.

Reply:
The file is removed; please resolve.